src/compas_timber/utils/__init__.py

```diff
@@ -291,6 +292,32 @@ def distance_segment_segment(segment_a, segment_b):
     return distance_point_point(pt_seg_a, pt_seg_b)
 
 
+def angle_vectors_projected(vector_1, vector_2, normal):
```

Comment:
how do you feel about naming these `vector_a` and `vector_b` instead? just for consistency with the other functions in this module.